Source/WebCore/css/CSSSelector.h

```diff
@@ -118,7 +118,11 @@ class CSSSelector {
     static bool isPseudoElementEnabled(PseudoElement, StringView, const CSSSelectorParserContext&);
     static std::optional<PseudoElement> parsePseudoElement(StringView, const CSSSelectorParserContext&);
     static std::optional<PseudoId> parseStandalonePseudoElement(StringView, const CSSSelectorParserContext&);
+    static bool pseudoClassRequiresArgument(PseudoClass);
+    static bool pseudoElementRequiresArgument(PseudoElement);
 
+    static bool hasCustomSelectorText(PseudoClass);
```

Comment:
This function name is rather vague. How about something like `pseudoClassMayHaveArgument`?

Reply:
Good idea, renamed!